taipy/core/notification/event.py

```diff
@@ -62,23 +62,20 @@ class EventEntityType(_ReprEnum):
     JOB = 6
     SUBMISSION = 7
 
-
-_NO_ATTRIBUTE_NAME_OPERATIONS = {EventOperation.CREATION, EventOperation.DELETION, EventOperation.SUBMISSION}
-_UNSUBMITTABLE_ENTITY_TYPES = (
-    EventEntityType.CYCLE,
-    EventEntityType.DATA_NODE,
-    EventEntityType.JOB,
-    EventEntityType.SUBMISSION,
-)
-_ENTITY_TO_EVENT_ENTITY_TYPE = {
-    "scenario": EventEntityType.SCENARIO,
-    "sequence": EventEntityType.SEQUENCE,
-    "task": EventEntityType.TASK,
-    "data": EventEntityType.DATA_NODE,
-    "job": EventEntityType.JOB,
-    "cycle": EventEntityType.CYCLE,
-    "submission": EventEntityType.SUBMISSION,
-}
```

Comment:
These constants are moved so that they become class variables, which can be edit properly by enterprise.

If they stay outside of the class, importing these constants and using them may not have the edit from enterprise.